deploy/helm/flightctl/templates/flightctl-default-user-roles.yaml

```diff
@@ -0,0 +1,95 @@
+{{ if eq .Values.global.target "acm" }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: flightctl-admin
```

Comment:
The `flightctl-` prefix on the roles looks a bit looks a bit verbose, but guess we'll need it in K8s to avoid collisions. :/

Reply:
+1